docs/docs_general/docs_whatsnew.dita

```diff
@@ -20,7 +20,7 @@ translate="no">ODF FlashSystem driver</ph>.</shortdesc>
 <entry colname="col1">31 January 2023</entry>
 <entry colname="col2">
 <p>The version information was added in IBM Documentation as part of the initial <ph translate="no"
->ODF FlashSystem driver</ph> 1.3.0 release.</p>
+>ODF FlashSystem driver</ph> 1.4.0 release.</p>
```

Comment:
I think it is better to update the doc files altogether, and not just the version. Now we just have incorrect docs (1.4.0 release date, 1.4.0 does not add support for multiple storage systems, etc.)

Reply:
reverting documentation files